src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -150,25 +144,12 @@ export default {
 		};
 	},
 	apollo: {
-		query: userInfoQuery,
 		preFetch(config, client) {
-			return Promise.all([
-				client.query({ query: experimentAssignmentQuery, variables: { id: EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: CATEGORIES_REDIRECT_EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: FLSS_ONGOING_EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: RELENDING_EXP_KEY } }),
-				client.query({ query: experimentAssignmentQuery, variables: { id: RECOMMENDED_REPLACEMENT_EXP_KEY } }),
-			]).then(() => {
-				const userInfoPromise = client.query({
-					query: userInfoQuery,
-				});
-
-				const recommendedLoansPromise = client.query({
-					query: flssLoansQueryExtended,
-					variables: prefetchedRecommendedLoansVariables
-				});
-
+			return client.query({
+				query: userInfoQuery
+			}).then(() => {
+				return client.query({ query: experimentAssignmentQuery, variables: { id: RECOMMENDED_REPLACEMENT_EXP_KEY } }); // eslint-disable-line max-len
```

Comment:
Can this query be ran with the `userInfoQuery` above?

Reply:
Oh yeah, this is a nice suggestion!